test/server/routes/oauth.test.ts

```diff
@@ -46,7 +46,6 @@ describe('server/routes/oauth', () => {
     const authorizeParams = new URLSearchParams({
       response_type: 'code',
       client_id: application.clientId,
-      client_secret: application.clientSecret,
```

Comment:
`client_secret` should never be part of the `authorize` URL params, since that would leak the client secret to intermediaries and the end-user.